com.github.alecaddd.sequeler.json

```diff
@@ -178,7 +178,7 @@
       "config-opts": ["--buildtype=release"],
       "sources": [{
         "type": "archive",
-        "url": "https://github.com/Alecaddd/sequeler/archive/v0.5.5.tar.gz",
+        "url": "https://github.com/Alecaddd/sequeler/archive/v0.5.7.tar.gz",
         "sha256": "ac5cd238bffe3afaf1c931009f7737cf0f22ef072a5b6b977149b38f4f20782b"
```

Comment:
You should update the sha256 too

Reply:
Thanks, how do I do that? Where do I grab that value? 😅

Reply:
Run `sha256sum $YOUR_FILENAME` on the command line and copy the value.